python/conditions.md

```diff
@@ -189,21 +189,21 @@ It’s conventional not to explicitly compare a condition to `True`, because the
 
 ```python
 >>> if ('Pineapple' in pizzas) == True:
-...     print 'Weird.'
+...     (print 'Weird.')
```

Comment:
Fix

Reply:
This is telling people not to compare to true and use this instead

```python
if 'Pineapple' in pizzas:
```